crt_portal/cts_forms/model_variables.py

```diff
@@ -254,11 +256,17 @@
 )
 
 # CRT views only
+READY_STATUS = 'ready'
+APPROVED_STATUS = 'approved'
+IN_REVIEW_STATUS = 'in_review'
+REJECTED_STATUS = 'rejected'
```

Comment:
this is redefining "REJECTED_STATUS" from above

consider prefixing these with BATCH_READY_STATUS, etc